p751toolbox/curve.go

```diff
@@ -226,6 +239,76 @@ func (xQ *ProjectivePrimeFieldPoint) Double(xP *ProjectivePrimeFieldPoint, aPlus
 	return xQ
 }
 
+// Given xP = x(P), xQ = x(Q), and xPmQ = x(P-Q), compute xPaddQ = x(P+Q) and  x2P = x(2P).
+func xDblAdd(curve *CachedCurveParameters, xP, xQ, xPmQ *ProjectivePoint) (x2P, xPaddQ ProjectivePoint) {
```

Comment:
I think that should probably be a method of *CachedCurveParameters

Reply:
Since this function operates over points, thus now it is a method of `ProjectivePoint` and `ProjectivePrimeFieldPoint`.
Good suggestion.